pkg/networkservice/common/connect/server_test.go

```diff
@@ -91,27 +128,31 @@ func TestConnectServer_Request(t *testing.T) {
 
 		urlA := &url.URL{Scheme: "tcp", Host: "127.0.0.1:10000"}
 		serverA := new(captureServer)
-
-		err := startServer(ctx, urlA, next.NewNetworkServiceServer(
+		nsServerA := next.NewNetworkServiceServer(
 			serverA,
 			newEditServer("a", "A", &networkservice.Mechanism{
 				Cls:  cls.LOCAL,
 				Type: kernel.MECHANISM,
 			}),
-		))
+		)
+		err := startServer(ctx, urlA, nsServerA)
 		require.NoError(t, err)
+		require.NoError(t, checkServer(urlA, nsServerA))
 
 		urlB := &url.URL{Scheme: "tcp", Host: "127.0.0.1:10001"}
 		serverB := new(captureServer)
-
-		err = startServer(ctx, urlB, next.NewNetworkServiceServer(
+		nsServerB := next.NewNetworkServiceServer(
 			serverB,
 			newEditServer("b", "B", &networkservice.Mechanism{
 				Cls:  cls.LOCAL,
 				Type: memif.MECHANISM,
 			}),
-		))
+		)
+		err = startServer(ctx, urlB, nsServerB)
 		require.NoError(t, err)
+		require.NoError(t, checkServer(urlB, nsServerB))
+
+		ignoreCurrentGoroutines := goleak.IgnoreCurrent()
```

Comment:
Can be replace with `defer goleak.VerifyNone(t, goleak.IgnoreCurrent())` since it is a function call.

Reply:
Done